Assets/scss/components/_project-leads.scss

```diff
@@ -0,0 +1,85 @@
+.project-leads-component {
+    margin: toRem(20);
+    padding: toRem(20);
+    border: toRem(1) solid var(--very-pale-shade-gray);
+    border-radius: toRem(8);
+    box-shadow: 0 toRem(2) toRem(4) rgba(0, 0, 0, 0.1);
+
+    h2 {
```

Comment:
@dkdeepesh  target this with class name instead if tag name.
Cc:-   @mathrik  @jvaishnavrp  @msharmaRP